atd-vzd/migrations/default/1715960023005_update_recommendations_and_notes_fks/up.sql

```diff
@@ -0,0 +1,21 @@
+alter table recommendations rename column crash_id to atd_txdot_crashes_crash_id;
+
+alter table recommendations
+    add column crash_id integer,
+    add constraint recommendations_crashes_id_fkey foreign key (
+        crash_id
+    ) references crashes (id),
+    drop constraint recommendations_crash_id_key,
+    add constraint recommendations_crash_id_key unique (crash_id),
+    alter column atd_txdot_crashes_crash_id drop not null;
```

Comment:
@johnclary do i need to add the updates i made to this migration to the `sql_templates` files?

Reply:
yes, please!

Reply:
cool just updated that!